### PR TITLE
Add S6 Import/Export test and remove old NetworkLoadSave test

### PR DESCRIPTION
**Pull request by ZehMatt**
This one should be much better to verify that the import/export works. I've also created a park that contains at least one ride of every category, bpb is missing quite a few things.

### CODE REVIEW

**Comment by duncanspumpkin** [comment]
Could also do with a td6 import export

File: test/tests/S6ImportExportTests.cpp [hunk 1]
```diff
@@ -0,0 +1,570 @@
...
+static void AdvanceGameTicks(uint32_t ticks, std::unique_ptr<IContext>& context)
+{
+    auto* gameState = context->GetGameState();
+    for (uint32_t i = 0; i < ticks; i++)
+    {
+        gameState->UpdateLogic();
+    }
+}
+
+#define COMPARE_FIELD(field) ASSERT_EQ(left.field, right.field)
```
Comment (janisozaur) [inline test/tests/S6ImportExportTests.cpp]: Why assert? The test could continue to find all mismatched fields with expect

Comment (ZehMatt) [reply test/tests/S6ImportExportTests.cpp]: Can change it to EXPECT_EQ

**Comment by ZehMatt** [comment]
@duncanspumpkin Do you mean a separate test?

**Comment by duncanspumpkin** [comment]
Yeah. Was more a note to future self to do it

**Comment by ZehMatt** [comment]
Just pushed a temporary commit, seems to work nicely.

**Comment by duncanspumpkin** [comment]
This okay to merge now?

**Comment by ZehMatt** [comment]
No objections on my end.

**Pull request merged**